src/status_im/contexts/communities/actions/addresses_for_permissions/events.cljs

```diff
@@ -0,0 +1,58 @@
+(ns status-im.contexts.communities.actions.addresses-for-permissions.events
+  (:require
+    [schema.core :as schema]
+    [taoensso.timbre :as log]
+    [utils.re-frame :as rf]))
+
+(defn get-permissioned-balances
+  [{[event-id _] :original-event} [community-id]]
```

Comment:
Note: this is a nice way to grab the current event ID. Often the case when we want to log a descriptive error.